llvm/test/CodeGen/X86/huge-stack-offset.ll

```diff
@@ -1,23 +1,15 @@
-; RUN: llc < %s -mtriple=x86_64-linux-unknown -verify-machineinstrs | FileCheck %s --check-prefix=CHECK-64
-; RUN: llc < %s -mtriple=i386-linux-unknown -verify-machineinstrs | FileCheck %s --check-prefix=CHECK-32
+; RUN: llc < %s -mtriple=x86_64-linux-unknown -verify-machineinstrs | FileCheck %s
```

Comment:
Don't need verify-machineinstrs

Reply:
Why have you dropped 32-bit test coverage?

Reply:
`-verify-machineinstrs` caught a few issues the last time I tried to merge this so i'd prefer to keep it unless there's a strong reason not to.

This test tries to stack allocate ~5gb which isn't valid on i386 so it wasn't clear to me what the point of the test was. I can add it back if you'd prefer.

Reply:
Just because it is invalid or won't work doesn't mean it's not worth testing. Ideally would emit an error or something, but the behavior should be captured in a test

Reply:
That's fair. I think emitting an error would be good so I might take a stab at doing that in another PR. For now, I've reverted this.